### DISCUSSION TITLE
use GQty helper functions

### DISCUSSION THREAD

**Discussion by roeean** [wpengine/faustjs#1040] [Q&A] [answered]
Hi,

In your documentation, it is indicated that you can use **GQty's helpers**, which are very necessary in some cases.
link: https://faustjs.org/docs/gqty-intro

I'm trying to use **selectFields()** in my next.js project to define specific recursive depth (https://gqty.dev/docs/client/helper-functions#selectfields). Still, unfortunately, I can't access it (nor any other helpers) because they don't exist - neither in **client** nor in **client.client**.

How can access them?

Thanks in advanced, 

Roee Angel

**Comment by theodesp** [comment]
Hey @roeean. We do not expose them directly. You will have to include `gqty` as a dependency to do that.

**Comment by theodesp** [comment, marked as answer]
You can see an example in this blueprint code:
https://github.com/wpengine/atlas-blueprint-blog/blob/dadab61b51a1cc28bf3ef9cc284a3571269fcb79/src/hooks/useSearch.js#L3
